cmd/fyne/internal/mobile/build_androidapp.go

```diff
@@ -76,10 +76,12 @@ func goAndroidBuild(pkg *packages.Package, bundleID string, androidArchs []strin
 		if err := mkdir(filepath.Dir(libAbsPath)); err != nil {
 			return nil, err
 		}
+		/*
```

Comment:
This isn't how we format comments - just a line starting "//" uses much less space.
Is it possible to use something that is clearer English, perhaps:
"If building release and no ldflags are set then remove the useless debug and DWARF build options"